pkg/operator/ceph/object/objectstore.go

```diff
@@ -591,22 +591,31 @@ func deletePools(context *Context, spec cephv1.ObjectStoreSpec, lastStore bool)
 		pools = append(pools, rootPool)
 	}
 
-	var wg sync.WaitGroup
-	for _, pool := range pools {
-		wg.Add(1)
-		name := poolName(context.Name, pool)
-		go func() {
-			defer wg.Done()
+	if configurePoolsConcurrently() {
+		var wg sync.WaitGroup
```

Comment:
Suggestion:  use `errgroup` to wait for completion of all the goroutines.  Similar to how its used while creating the pools.

Reply:
Done.

Reply:
this is resolved.